server/validators/postDiaryEntryValidator/titleValidator.js

```diff
@@ -1,10 +1,19 @@
 import validator from 'validator';
 
-const postDiaryEntryTitleValidator = (data) => {
+const postDiaryEntryTitleValidator = (data, isEditing, diaryEntry) => {
```

Comment:
Nice work you did here. Making adding an additional parameter to check if it's a `PUT` or `POST` request

Reply:
Thank you @torkpe for your feedback and help.